README.md

```diff
@@ -16,10 +16,10 @@ and more...
 
 As a learner in social sciences, it is really the time to study computational methods.
 
-##Things I have learned so far
+## Things I have learned so far
 How to use Git workflow. I also found `<fork>` an interesting term. You may find [this website](https://www.atlassian.com/git/tutorials/comparing-workflows) useful for understanding `<fork>`.
```

Comment:
This website looks really useful! Like you, I am familiar with R but not so much with github. To be honest, everything still feels quite unintuitive to me, so the link is very much appreciated!